tests/unit/test_client.py

```diff
@@ -3627,29 +3633,7 @@ def test_create_job_copy_config(self):
             }
         }
 
-        self._create_job_helper(
-            configuration, "google.cloud.bigquery.client.Client.copy_table",
-        )
-
-    def test_create_job_copy_config_w_single_source(self):
```

Comment:
Ditto on this. I see no reason to remove this test, as it's still documented as a possibility in the API.